src/app/feedback-form/feedback-form.component.html

```diff
@@ -18,9 +21,22 @@ <h3 class="contact-form__title">Get in Touch with Us!</h3>
           class="contact-form__input"
           type="email"
           id="email"
-          name="email"
-          required
+          formControlName="email"
         />
+        <div
+          class="error-message"
+          [hidden]="
+            !(form.get('email')?.touched && form.get('email')?.hasError('required') && !form.get('email')?.value)
```

Comment:
you can use @let to save the variable form.get('email) so you dont have to keep repeating it

Reply:
@Khousheish I tried to use @let in .html file, but just received errors. Can you please help me?

<img width="950" alt="Screenshot 2024-10-24 at 18 08 52" src="https://github.com/user-attachments/assets/06092285-4981-4c32-9d04-38de517e1b6a">

Reply:
I tried to do some research, and it may be that @let is not supported in our template

Reply:
oh looks like on the angular version you have that was not added yet 😅 Thats unfortunate.
Can you use *ngrx-let?